modules/core/shared/src/main/scala/lucuma/core/model/Brightness.scala

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats._
+import lucuma.core.enum.WavelengthBand
+import lucuma.core.enum.BrightnessUnits
+import lucuma.core.math.BrightnessValue
+import monocle.Focus
+import monocle.Lens
+
+/**
+ * Describes the brightness of a target on a given band
+ */
+final case class Brightness(
+  value:  BrightnessValue,
+  band:   WavelengthBand,
+  error:  Option[BrightnessValue],
+  system: BrightnessUnits
```

Comment:
> Should this be units?

Thanks for catching that.

> Should we consider that this can also be SurfaceBrightness?

Yes, I'm planning to tackle that in a future PR.